locations/spiders/rubios.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+
+from locations.items import GeojsonPointItem
+
+
+class RubiosSpider(scrapy.Spider):
+    name = "rubios"
+    allowed_domains = ['rubios.com']
+    start_urls = (
+        'https://www.rubios.com/sitemap.xml',
+    )
+
+    def parse(self, response):
+        response.selector.remove_namespaces()
+        city_urls = response.xpath('//url/loc/text()').extract()
+        for path in city_urls:
+            regex = re.compile(r'http\S+rubios.com/store-locations/\S+/\S+/\S+')
+            if not re.search(regex,path):
```

Comment:
Instead of checking the negative condition here, only `yield scrapy.Request()` if there's a match.

Reply:
I apologize, I just saw these changes. Will adjust.